substrate/client/rpc-spec-v2/src/chain_head/chain_head_follow.rs

```diff
@@ -71,6 +69,8 @@ pub struct ChainHeadFollower<BE: Backend<Block>, Block: BlockT, Client> {
 	current_best_block: Option<Block::Hash>,
 	/// LRU cache of pruned blocks.
 	pruned_blocks: LruMap<Block::Hash, ()>,
```

Comment:
Believe that `pruned_blocks` are no longer needed (https://github.com/paritytech/polkadot-sdk/issues/5605), will tackle this in another PR